tests/Fixtures/TestController.php

```diff
@@ -608,6 +607,116 @@ public function withInjectedEnumAndModel(Category $category, TestUser $user)
         return null;
     }
      */
+
+    /**
+     * @example
```

Comment:
I think I'd rather change this to two separate tags, `@exampleRequest` and `@exampleResponse`. My reasoning is that `@example` could be a generic DocBlock tag, and I'd prefer to avoid any conflicts.